trieste/acquisition/function.py

```diff
@@ -137,9 +137,9 @@ def expected_improvement(model: ProbabilisticModel, eta: tf.Tensor, at: QueryPoi
     return (eta - mean) * normal.cdf(eta) + variance * normal.prob(eta)
 
 
-class MaxValueEntropySearch(SingleModelAcquisitionBuilder):
+class MinValueEntropySearch(SingleModelAcquisitionBuilder):
     """
-    Builder for the max-value entropy search acquisition function (for function minimisation)
+    Builder for the min-value entropy search acquisition function (for function minimisation)
```

Comment:
the only thing remaining afaict is that it would be good to mention this is Max-value Entropy Search for objective minimization (unless calling it min-value entropy search is standard in the literature)

Reply:
Yeah that a good idea. I've changed it.